### PR TITLE
Update Readme file

### PR DESCRIPTION
**Pull request by lonelyCZ**
Signed-off-by: lonelyCZ <531187475@qq.com>

### CODE REVIEW

File: cm-vault/README.md [hunk 1]
```diff
@@ -45,25 +45,24 @@ You can configurate custom variables in `common.tfvars`.
 
 ```
 # Path to the kubeconfig file to use for connecting kubernetes cluster. Default is "~/.kube/config"
-kubeconfig_path="~/.kube/config"
+kubeconfig_path = "~/.kube/config"
+
+# Cert-manger version to install. Default is "v1.8.2"
+cm_version = "v1.8.2"
```
Comment (irbekrm) [inline cm-vault/README.md:51]: Note: we'll probably want to bump the default to v1.9.0 that was released last Friday https://github.com/cert-manager/cert-manager/releases

**Review by irbekrm — APPROVED** [review]
Thanks @lonelyCZ this looks good!

/lgtm

File: cm-vault/README.md (hunk 1 above)
Comment (lonelyCZ) [reply cm-vault/README.md:51]: Ok, I will do it.

**Pull request merged**